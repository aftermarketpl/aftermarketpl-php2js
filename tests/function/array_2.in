$a = array(1, 2, 3);
function f_array2($b)
{
    $b[1] = 4;
}
f_array2($a);
return $a[1];
