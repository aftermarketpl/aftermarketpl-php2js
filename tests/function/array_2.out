var a;
a = [
    1,
    2,
    3
];
function f_array2(b)
{
    b = b.slice(0);
    b[1] = 4;
}
f_array2(a);
return a[1];
